dev/devicelab/manifest.yaml

```diff
@@ -489,6 +489,13 @@ tasks:
     required_agent_capabilities: ["mac/ios"]
     timeout_in_minutes: 20
 
+  system_debug_ios:
+    description: >
+      Tests that the Engine correctly initializes the system debugger for debug-mode iOS apps.
+    stage: devicelab_ios
+    required_agent_capabilities: ["mac/ios"]
+    timeout_in_minutes: 10
```

Comment:
All new devicelab tests should be marked as `flaky: true` for at several commits to ensure they work as expected.

Reply:
Done.